src/containers/WalletHoc/reducer.js

```diff
@@ -60,6 +62,12 @@ abiDecoder.addABI(ERC20ABI);
 
 function walletHocReducer(state = initialState, action) {
   switch (action.type) {
+    case DRAG_WALLET:
+      return state
+        .set(
+          'wallets',
+          fromJS(arrayMove(state.get('wallets').toJS(), action.oldIndex, action.newIndex))
```

Comment:
This operation may need a bit more care, since it can affect the wallet's persistence data in the localStorage. Perhaps it needs some tests to confirm the `arrayMove` is always working as expected. 

What happens when the `action.oldIndex` or `action.newIndex` comes with unexpected values such as `null`?

Reply:
Very good point
I'll create our own `arrayMove` that we can do better validation & checks on